File: bin/Debug/net5.0/v4.4.2/lgcy-on-lgcy/examples/t_api-captcha-mvc4-web.security.membership/aspx/csharp/App_Code/CaptchaHelper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

using BotDetect.Web.Mvc;

namespace Mvc40CaptchaExampleAspxCSharp.App_Code
{
    public static class CaptchaHelper
    {
        public static MvcCaptcha GetRegistrationCaptcha()
        {
            // create the control instance
            MvcCaptcha registrationCaptcha = new MvcCaptcha("RegistrationCaptcha");

            // set up client-side processing of the Captcha code input textbox
            registrationCaptcha.UserInputID = "CaptchaCode";

            // Captcha settings
            registrationCaptcha.ImageSize = new System.Drawing.Size(200, 50);
            registrationCaptcha.CodeLength = 4;

            return registrationCaptcha;
        }

        public static MvcCaptcha GetLoginCaptcha()
        {
            // create the control instance
            MvcCaptcha loginCaptcha = new MvcCaptcha("LoginCaptcha");

            // set up client-side processing of the Captcha code input textbox
            loginCaptcha.UserInputID = "CaptchaCode";

            // Captcha settings
            loginCaptcha.ImageSize = new System.Drawing.Size(200, 50);
            loginCaptcha.CodeLength = 4;

            return loginCaptcha;
        }
    }
}

File: bin/Debug/net5.0/v4.4.2/lgcy-on-lgcy/examples/t_api-captcha-mvc4-web.security.membership/razor/vbnet/App_Code/CaptchaHelper.vb
Imports BotDetect.Web.Mvc

Public Class CaptchaHelper

    Public Shared Function GetRegistrationCaptcha() As MvcCaptcha

        ' create the control instance
        Dim registrationCaptcha As MvcCaptcha = New MvcCaptcha("RegistrationCaptcha")

        ' set up client-side processing of the Captcha code input textbox
        registrationCaptcha.UserInputID = "CaptchaCode"

        ' Captcha settings
        registrationCaptcha.ImageSize = New System.Drawing.Size(200, 50)
        registrationCaptcha.CodeLength = 4

        Return registrationCaptcha

    End Function

    Public Shared Function GetLoginCaptcha() As MvcCaptcha

        ' create the control instance
        Dim loginCaptcha As MvcCaptcha = New MvcCaptcha("LoginCaptcha")

        ' set up client-side processing of the Captcha code input textbox
        loginCaptcha.UserInputID = "CaptchaCode"

        ' Captcha settings
        loginCaptcha.ImageSize = New System.Drawing.Size(200, 50)
        loginCaptcha.CodeLength = 4

        Return loginCaptcha

    End Function
End Class
